### DISCUSSION TITLE
About mangle in CLI

### DISCUSSION THREAD

**Discussion by ntnguyen1234** [j4k0xb/webcrack#108] [Q&A]
Sorry, are there any documentations about how to use `mangle` in CLI version? I access https://webcrack.netlify.app/docs/guide/cli.html but there's nothing there mentioning how to pass mangle names. For example, the web version only says `Short names`..., how can I use `Short names` in CLI version?

**Comment by j4k0xb** [comment]
it's only possible via the playground and js api at the moment
I might have to add another CLI option to avoid a breaking change

**Reply by ntnguyen1234** [reply]
Thanks for your information.
